spec/datadog/tracing/distributed/b3_single_spec.rb

```diff
@@ -29,7 +27,7 @@
         )
       end
 
-      it { is_expected.to eq(Datadog::Tracing::Distributed::Headers::Ext::B3_HEADER_SINGLE => '2710-4e20') }
+      it { is_expected.to eq('b3' => '2710-4e20') }
```

Comment:
In other cases in this PR, I've very much agreed with inlining the actual header names, because `x-datadog-potato` is reasonably obvious what it is.

Here.... I'm way less sure. Can I convince you to add a `let(:b3_single_header) { 'b3' }` and use that instead? It seems to be me that it would help with readibility :)

Reply:
Sure, that seems fine!

Reply:
Done for b3 single header.
I was going to do b3 multi header, but those actually read nicely like:
`x-b3-traceid`
`x-b3-spanid`
`x-b3-sampled`

Do you think the b3 multi header would benefit from it as well?

Reply:
Nah, I think single header is the only one that's confusing. I think the others are specific enough to be obvious in context (and one can even quickly grep/google them to find out what they are)